packages/manager/modules/pci/src/projects/project/private-registry/list/list.html

```diff
@@ -0,0 +1,70 @@
+<div>
+    <oui-page-header
+        data-heading="{{ 'private_registry_title' | translate}}">
+        <!-- guide menu -->
+        <oui-guide-menu data-text="{{ ::'private_registry_guide_main' | translate }}" data-ng-if="false">
```

Comment:
```suggestion
        <oui-guide-menu data-text="{{ ::'private_registry_guide_main' | translate }}">
```
Will not be displayed if not